profiles/common/modules/features/cce_basic_config/cce_basic_config.features.filter.inc

```diff
@@ -137,6 +137,39 @@ function cce_basic_config_filter_default_formats() {
     ),
   );
 
+  // Exported format: Full HTML + Change tracking.
```

Comment:
Shouldn't this new format contain each filter that the full html contains as the name suggests? Then looks like it's missing 'toolbox_sanitize' and 'filter_tokens'.